scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -410,7 +410,7 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
      */
     internal fun getClosestCopyrightStatements(
         path: String,
-        copyrights: JsonNode,
+        copyrights: List<JsonNode>,
```

Comment:
Please use 'Collection' as type to not specialize this without need.

Reply:
Done.